Alpaca.Markets/RestClient.cs

```diff
@@ -26,6 +28,12 @@ public sealed partial class RestClient : IDisposable
 
         private readonly String _polygonApiKey;
 
+        private readonly IEnumerable<Int32> _retryHttpStatuses;
+
+        private readonly String _alpacaApiVersion;
+
+        private static readonly IEnumerable<String> _supportedApiVersions = new[] { "1", "2" };
```

Comment:
Performance issue - same as for _retryHttpStatuses field.

Reply:
Again, switched to HashSet.